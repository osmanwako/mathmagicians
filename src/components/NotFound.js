function NotFound() {
  return (
    <p>
      Page is not found. 404!
    </p>
  );
}
export default NotFound;
