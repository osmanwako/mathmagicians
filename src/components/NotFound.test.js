import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

test('NotFound component renders correctly', () => {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
