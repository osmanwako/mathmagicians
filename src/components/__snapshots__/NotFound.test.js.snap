// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders correctly 1`] = `
<p>
  Page is not found. 404!
</p>
`;
